src/measure_theory/borel_space.lean

```diff
@@ -478,4 +487,82 @@ begin
   { simp [measurable_const] }
 end
 
+lemma measurable_of_real : measurable ennreal.of_real :=
+measurable_of_continuous ennreal.continuous_of_real
+
 end ennreal
+
+namespace measure_theory
+
+open topological_space
+
+lemma measurable_smul' {α : Type*} {β : Type*} {γ : Type*}
+  [semiring α] [topological_space α] [second_countable_topology α]
+  [topological_space β] [add_comm_monoid β] [second_countable_topology β]
+  [semimodule α β] [topological_semimodule α β] [measurable_space γ]
+  {f : γ → α} {g : γ → β} (hf : measurable f) (hg : measurable g) :
+  measurable (λ c, f c • g c) :=
+measurable_of_continuous2 (continuous_smul continuous_fst continuous_snd) hf hg
+
+lemma measurable_smul {α : Type*} {β : Type*} {γ : Type*}
+  [semiring α] [topological_space α] [second_countable_topology α]
+  [topological_space β] [add_comm_monoid β] [second_countable_topology β]
+  [semimodule α β] [topological_semimodule α β] [measurable_space γ]
+  {c : α} {g : γ → β} (hg : measurable g) : measurable (λ x, c • g x) :=
+measurable.comp (measurable_of_continuous (continuous_smul continuous_const continuous_id)) hg
+
+lemma measurable_dist' {α : Type*} [metric_space α] [second_countable_topology α] :
+  measurable (λp:α×α, dist p.1 p.2) :=
+begin
+  rw [borel_prod], apply measurable_of_continuous,
```

Comment:
the style is weird here: why is the apply on the same line, and why is there such a big indentation on the next line? These comments apply also to the next 30 lines or so.